### PR TITLE
Mobile v2.8 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by enahum — APPROVED** [review]

File: source/deploy/mobile-app-changelog.md [hunk 1]
```diff
@@ -2,12 +2,38 @@
...
+### Compatibility
+ - **Upgrade to server version v7.8.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html) (ESR) v7.1.0 has ended and upgrading to server ESR v7.8.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Improvements
+ - Calls: Added incoming call notifications for Direct Message and Group Message calls.
+ - Added some performance improvements.
+ - Removed unneeded requests to the server with malformed mentions.
+ - Adds a snackbar info when the ``Copy Info`` button is clicked on the **About** screen.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:25]:
```suggestion
 - Adds snackbar info when the ``Copy Info`` button is tapped on the **About** screen.
```

Comment (cwarnermm) [reply source/deploy/mobile-app-changelog.md:25]: what is snackbar information? (that's a new term for me!)

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline

**Review by larkox — APPROVED** [review]

File: source/deploy/mobile-app-changelog.md (hunk 1 above)
Comment (amyblais) [reply source/deploy/mobile-app-changelog.md]: It came from the release note in this PR https://github.com/mattermost/mattermost-mobile/pull/7488. I'll change it to "toast".

**Review by streamer45 — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**